PythonRegression/tests/features/steps/api_test_steps.py

```diff
@@ -13,52 +14,98 @@
 neighbors = static_vals.TEST_NEIGHBORS
 testAddress = static_vals.TEST_ADDRESS
 
+localVar = "TESTINGLOCALVARIABLESAAAAAAAAAAAAAAAAAAAAAAAAAAH"
+
 config = {}
 responses = {'getNodeInfo':{},'getNeighbors':{},'getTips':{},'getTransactionsToApprove': {},'getTrytes':{}}   
```

Comment:
So those are variable that are passed between steps?
If so how do you clean them up?
Aren't you suppose to use the `world` variable?

Reply:
This will be done in a different PR